src/Tizen.NUI/src/public/BaseComponents/RiveAnimationView.cs

```diff
@@ -0,0 +1,161 @@
+/*
+* Copyright(c) 2021 Samsung Electronics Co., Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Tizen.NUI.BaseComponents
+{
+    /// <summary>
+    /// RiveAnimationView renders an animated vector image (Rive file).
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
+    public class RiveAnimationView : View
+    {
+        static RiveAnimationView() { }
+
+        /// <summary>
+        /// RiveAnimationView constructor.
+        /// </summary>
+        /// <param name="url">The rive resource URL</param>
+        /// <since_tizen> 9 </since_tizen>
+        public RiveAnimationView(string url) : this(Interop.RiveAnimationView.New(url), true)
```

Comment:
Is that the NUI policy? 
How to change the resource? if user wants to reset the resource...

Reply:
I will consider this next patch